doc/contributing/maintaining-shared-library-support.md

```diff
@@ -0,0 +1,119 @@
+# Maintaining shared library support
+
+Node.js unofficially supports a build option where Node.js is built as
+a shared library (node.dll/libnode.so). The shared library provides
+an easy way to embed Node.js into other applications and to
+have multiple applications use a single copy of Node.js instead
+of having to bundle in the full Node.js footprint into each application.
+For workloads that require multiple node instances this can result
```

Comment:
```suggestion
For workloads that require multiple Node.js instances this can result
```

Reply:
Reading further down, `node` might be correct. If it is, `node` as a concept should be introduced before it's used here.